XavierGuertin/WildfireVisualizationProject
# #116 - feat(PostgreSQL): Persist PostgreSQL Database in Docker

### Summary
PostgreSQL data persistence

### Changes Made
- Mounted a Docker volume to persist PostgreSQL data

### Testing Instructions
1. Build and run the app with Docker
2. Make changes to the database
3. Verify that the changes are saved within the project

### Screenshots
![image](https://github.com/user-attachments/assets/5202cf3c-3668-468d-b55f-1a18eb6f6474)

### Additional Notes
- I would recommend using pgAdmin to work with the database (shown in the screenshot)
